src/main/kotlin/no/nav/medlemskap/services/aareg/AaregMapper.kt

```diff
@@ -57,13 +57,12 @@ fun mapArbeidsForholdType(arbeidsforhold: AaRegArbeidsforhold): Arbeidsforholdst
             return Arbeidsforholdstype.ANDRE
         }
     }
-
 }
 
-//TODO --> LA INN NOR SOM DEAFULT FOR TEST
-fun mapArbeidsgiver(arbeidsforhold: AaRegArbeidsforhold): Arbeidsgiver {
-    return Arbeidsgiver(arbeidsforhold.type, arbeidsforhold.opplysningspliktig.organisasjonsnummer, "NOR");
-
+fun mapArbeidsgiver(arbeidsforhold: AaRegArbeidsforhold, arbeidsgiversLand: Map<String, String>): Arbeidsgiver {
+    val identifikator = arbeidsforhold.arbeidsgiver.organisasjonsnummer
+            ?: (arbeidsforhold.arbeidsgiver.offentligIdent ?: arbeidsforhold.arbeidsgiver.aktoerId)
+    return Arbeidsgiver(arbeidsforhold.type, arbeidsgiversLand[identifikator])
```

Comment:
her sender vi inn arbeidsforholdets type. Jeg er litt usikker på den, er det ikke evt arbeidsgivers type vi er ute etter? Merk at mappet ditt som heter "arbeidsgiversLand" vel bare inneholder land for personer, og enhetstype for organisasjoner. Så dataene du sender inn i Arbeidsgiver her er kanskje ikke helt på stell?

Reply:
Her må du forklare litt nærmere tror jeg. Ser ikke helt hva du mener.

Vi er ute etter hvilket land arbeidsgiver tilhører. Slik arbeidsforhold.type ser ut for meg, er at den viser om hvilken type arbeidsforhold det er snakk om; altså maritim, pilot, freelance, etc.

Anngående arbeidsgiversland map; helt enig. Vi skal ikke ta noen forhold til hvilket land det er ennå, det skulle bli gjort i reglene. Skal jeg da fjerne mappen, eller tenker vi refaktorisering?

Reply:
Men burde ikke arbeidsforholdets type mappes inn i arbeidsforhold, ikke inn i arbeidsgiver? Jeg ville trodd at arbeidsgiver.type var enhetstype? Altså om det er NUF eller ORG eller whatnot?

Reply:
Burde arbeidsgiver kanskje ha felter for a) om det er en person eller en org b) typen org, hvis org, c) land og d) ident ? (Litt usikker på om vi trenger d egentlig..)

Reply:
Tenker å gjøre noe med dette i en egen pull request